feature/experimental/policy/policy_base/README.md

```diff
@@ -0,0 +1,25 @@
+# RT-3.1: Policy based VRF selection
```

Comment:
Could we use the directory `feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection` for both files?  This was how #521 was done.

Reply:
Done, in new commit. Thank You.